packages/pyxis-react/src/components/Toggle/Toggle.tsx

```diff
@@ -0,0 +1,57 @@
+import React, {
+  FC,
+  ForwardedRef,
+  InputHTMLAttributes,
+  SyntheticEvent,
+  forwardRef,
+} from 'react';
+
+import { LabelProps } from 'components/Form/Label';
+import classNames from 'classnames';
+
+const getClasses = (disabled?: boolean) => classNames(
+  'toggle',
+  {
+    'toggle--disabled': disabled,
+  },
+);
```

Comment:
Aggiungiamo anche la `className` custom
```suggestion
const getClasses = (disabled?: boolean, className?: string) => classNames(
  'toggle',
  {
    'toggle--disabled': disabled,
  },
  className
);
```